README.rst

```diff
@@ -105,75 +117,166 @@ Actions overview
 
 The actions available are, in alphabetical order:
 
-- ``add``: Add a file to the queue for processing with generate_climos
-- ``alter-params``: Update entries with generate_climos params and PBS params (but not status). 
-  Updated entry must be in status NEW. 
-  WARNING: ANY entry that partially matches the input filename is updated.
-- ``list``: List entries in generate_climos queue
-- ``reset``: Reset the status of a queue entry
-- ``script``: Write to stdout the script that will be or was submitted by ``jq submit`` for
-  a specified queue entry.
-- ``submit``: Dequeue one or more generate_climos queue entries with NEW status,
-  and submit a PBS job for each, updating the queue entries accordingly.
-- ``update-email``: Update generate_climos queue using PBS status email
-- ``update-qstat``: Update generate_climos queue using PBS qstat
+=================== ============================================
+Action              Effect
+=================== ============================================
+``add``             Add a file to the queue for processing with ``generate_climos``
+
+``alter-params``    Update entries with generate_climos params and PBS params (but not status).
+                    Updated entry must be in status NEW.
+
+                    WARNING: ANY entry that partially matches the input filename is updated.
+
+``hold``            Set entries in ``generate_climos`` queue from NEW to HOLD status
+
+``unhold``          Set entries in ``generate_climos`` queue from HOLD to NEW status
+
+``list``            List entries in ``generate_climos`` queue
+
+``reset``           Reset the status of a queue entry
+
+``script``          Write to stdout the script that will be or was submitted by ``jq submit`` for
+                    a specified queue entry.
+
+``submit``          Dequeue one or more ``generate_climos`` queue entries with NEW status,
+                    and submit a PBS job for each, updating the queue entries accordingly.
+
+``summarize``       Summarize entries in ``generate_climos`` queue (generate statistics)
+
+``update-email``    Update ``generate_climos`` queue using PBS status email
```

Comment:
update-email baffled me as a user. Should we just remove this option? update-qstat works great.

Reply:
It's useful if you waited too long to do an update-qstat. Eventually PBS dumps info and you can't get it back. But I agree it's not much use most of the time, plus it's laborious to use. Punishment for the tardy.